apps/testingaccessibility/src/components/pricing.tsx

```diff
@@ -196,7 +196,7 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
               userId,
               quantity,
               productId: formattedPrice?.id,
-              bulk: quantity > 1,
+              bulk: Boolean(formattedPrice?.bulk),
```

Comment:
There are several places where we are doing this `quantity > 1` check which can be inaccurate.